Xamarin.Essentials/Contacts/Contacts.uwp.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.ApplicationModel.Contacts;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Contacts
+    {
+        static async Task<Contact> PlatformPickContactAsync()
+        {
+            var contactPicker = new ContactPicker();
+
+            try
+            {
+                var contactSelected = await contactPicker.PickContactAsync();
+
+                if (contactSelected == null)
+                    return null;
+
+                var contactManager = await ContactManager.RequestStoreAsync();
+                var contact = await contactManager.FindContactsAsync(contactSelected.Name);
```

Comment:
Why do we need the ContactManager? We already have the contact.

Looking at the example in the docs, it appears they just use the contact, or optionally reduce the results by setting the fields: https://docs.microsoft.com/en-us/windows/uwp/contacts-and-calendar/selecting-contacts#complete-example-single-contact

Reply:
Tested works fine without the manager and finding it... changed.